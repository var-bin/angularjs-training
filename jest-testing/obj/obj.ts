// obj.ts

"use strict";

const data = {
  one: 1,
  two: 2,
  three: 3,
  arr: [1, 2, 3, 4]
};

export { data };
