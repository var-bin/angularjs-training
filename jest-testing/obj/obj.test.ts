// obj.test.ts

"use strict";

import { data } from "./obj";

test("The data object should contain {one: 1, two: 2}", () => {
  expect(data).toMatchSnapshot();
});
