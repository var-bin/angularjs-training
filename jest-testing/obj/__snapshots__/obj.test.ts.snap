// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The data object should contain at least {one: 1, two: 2} 1`] = `
Object {
  "arr": Array [
    1,
    2,
    3,
    4,
  ],
  "one": 1,
  "three": 3,
  "two": 2,
}
`;
